README.md

```diff
@@ -411,6 +411,25 @@ change the UUID.
 You can't. This is because historical bookings still refer to that prison.
 Disable it instead (see above).
 
+#### Merging two prisons
+
+If two formerly-separate parts of an estate now have the same booking team and
```

Comment:
We all know what "formerly-separate parts of an estate" means, but I'm not sure that would be the case with someone new coming to the team.